test/e2e/network/dns.go

```diff
@@ -559,11 +559,6 @@ var _ = common.SIGDescribe("DNS", func() {
 	})
 
 	ginkgo.It("should work with the pod containing more than 6 DNS search paths and longer than 256 search list characters", func(ctx context.Context) {
-		ginkgo.By("Getting the kube-dns IP")
-		svc, err := f.ClientSet.CoreV1().Services("kube-system").Get(ctx, "kube-dns", metav1.GetOptions{})
```

Comment:
@gjkim42 is ok to remove the Service ClusterIP for this test?
https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/2595-expanded-dns-config

if is not possible, we can just add a Skip if the service Get returns ErrNotFound

Reply:
> As the test uses `dnsPolicy: ClusterFirst` (as opposed to `dnsPolicy: None`), it does not need to specify `dnsConfig.nameservers`. Omitting `dnsConfig.nameservers` obviates the need to look up the service.

This makes sense. I think it is ok to remove it.